src/@next/components/atoms/AddNewTile/test.tsx

```diff
@@ -0,0 +1,15 @@
+import { mount } from "enzyme";
```

Comment:
I believe shallow should be sufficient for this use case. `shallow` is faster than `mount`